MetricHubV2.sln

```diff
@@ -1,25 +1,39 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
-VisualStudioVersion = 15.0.28010.0
-MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MetricHub.Entrypoint", "src\MetricHub.Entrypoint\MetricHub.Entrypoint.csproj", "{C700C278-1DAC-4BDD-A8D9-9F5246983D9D}"
+VisualStudioVersion = 15.0.26124.0
+MinimumVisualStudioVersion = 15.0.26124.0
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{F1E9F313-48CB-4E7A-9472-E580DB91BBB2}"

```

Comment:
What does this project ("src") refer to?

Reply:
Pretty sure src is the solution folder here since the project is add `./src/MetricHub.Entrypoint`